### PR TITLE
Add Fortran support

### LINKED ISSUE #16: Add Fortran support
Could fftw be configured without Fortran support or would this better be a separate feedstock?

### PR DESCRIPTION
**Pull request by egpbos**
Added Fortran compiler, enabled Fortran build of FFTW (disabled --disable-fortran flag) and added a test to the recipe to check whether the Fortran symbols are indeed present in the library (the double precision Fortran symbols start with `dfftw_` instead of just `fftw_`, so that's easy to check).

Note that I also included commit 14660a4, which was first added automatically in #40 by some bot to update the conda-forge setup.

Fixes #16

### CODE REVIEW

**Comment by egpbos** [comment]
Removed Windows support; as discussed in #16, initial Fortran support can be just for Linux/macOS.

**Comment by jakirkham** [comment]
> The log length has exceeded the limit of 4 MB (this usually means that the test suite is raising the same exception over and over).

ref: https://travis-ci.org/conda-forge/fftw-feedstock/jobs/397702726

**Comment by jakirkham** [comment]
Also there should not be `python` builds at all here. Looks like you are running into issue ( https://github.com/conda/conda-build/issues/2989 ) as well. Would add empty `host` and `run` sections to `requirements` and then request the bot re-render again. Hopefully should fix the issue.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -19,14 +19,19 @@ build:
 requirements:
   build:
     - python         # [win]
```
Comment (isuruf) [inline recipe/meta.yaml]: Remove this line

File: recipe/meta.yaml [hunk 2]
```diff
@@ -19,14 +19,19 @@ build:
...
-    - toolchain
+    - cmake          # [win]
     - perl 5.22.2.1  # [not win]
-    - cmake  # [win]
+    - {{ compiler('c') }}        # [not win]
+    - {{ compiler('fortran') }}  # [not win]
 
 test:
   requires:
     - python {{ os.environ.get('PY_VER', '') + '*' }}  # [win]
```
Comment (isuruf) [inline recipe/meta.yaml]: Remove the 2 lines above

File: recipe/meta.yaml [hunk 3]
```diff
@@ -19,14 +19,19 @@ build:
 requirements:
```
Comment (isuruf) [inline recipe/meta.yaml]: Remove Lines 14-17

File: recipe/meta.yaml (hunk 1 above)
Comment (egpbos) [reply recipe/meta.yaml]: I didn't add this, so I'm not sure if I should be the one to remove it in this PR. @jakirkham what do you think?

File: recipe/meta.yaml (hunk 2 above)
Comment (egpbos) [reply recipe/meta.yaml]: You mean the compilers? I think those are necessary. How would you do it?

File: recipe/meta.yaml (hunk 3 above)
Comment (egpbos) [reply recipe/meta.yaml]: I also didn't add that, it was already there.

File: recipe/meta.yaml (hunk 2 above)
Comment (isuruf) [reply recipe/meta.yaml]: Not the compilers. Line 28 and 29

**Comment by egpbos** [comment]
@jakirkham unfortunately just adding empty host and run sections didn't work.

**Comment by isuruf** [comment]
@conda-forge-admin, please rerender

**Comment by jakirkham** [comment]
I've tried to fix some compiler issues in `master` to hopefully make things easier here. Could you please try merging with `master`? Only worry about conflicts inside the `recipe` directory. The rest choose whatever is easiest as re-rendering with the bot should resolve them. Please let me know if you have any questions. Will review this later next week when I get back.

**Comment by egpbos** [comment]
Applying the edits suggested by @isuruf does allow the checks to pass. I will try to merge with master.

**Comment by egpbos** [comment]
@conda-forge-admin, please rerender

**Comment by egpbos** [comment]
It seems easier to just make a new PR than to update this one, did that here: #49

**Pull request closed without merging**